shell/composables/useFeatureFlag.ts

```diff
@@ -0,0 +1,21 @@
+import { computed } from 'vue';
```

Comment:
We might want to call this something different to avoid confusion with rancher feature flags

Reply:
I renamed this to `useRuntimeFlag`.. I'm open to suggestions.